ros2_control_demo_bringup/launch/test_joint_trajectory_position_controller.launch.py

```diff
@@ -32,8 +32,8 @@ def generate_launch_description():
         [
             Node(
                 package="ros2_control_test_nodes",
-                executable="publisher_joint_trajectory_controller",
-                name="publisher_joint_trajectory_controller",
+                executable="publisher_joint_trajectory_position_controller",
```

Comment:
We should here switch to `ros2_controllers_test_nodes` from ros2_controllers repository. Can you do that, please?